projects/plugins/jetpack/extensions/store/membership-products/test/actions-test.js

```diff
@@ -172,39 +183,31 @@ describe( 'Membership Products Actions', () => {
         expect( getMessageMock ).toBeCalledWith( 'product requires a valid price', paymentPlanProductType );
     });
 
-    /*test( 'Happy case displays a success notice.', async () => {
+	test( 'Happy case displays a success notice.', async () => {
         // Given
         const anyValidProduct = buildAnyValidProduct();
         const paymentPlanProductType = PRODUCT_TYPE_PAYMENT_PLAN;
         const selectedProductCallback = jest.fn(() => {} );
-        const apiResponseProduct = {
-            id: 1,
-            title: 'anyTitle',
-            interval: 'anyInterval',
-            price: '12',
-            currency: 'anyCurrency',
-        }
         const registryProductList = [ apiResponseProduct, apiResponseProduct ];
         const registry = {
             select: () => ( { getProducts: () => registryProductList } )
         };
         const dispatch = jest.fn( () => {} );
-        const apiFetchMock = jest.spyOn( api, 'apiFetch' ).mockImplementation( () => apiResponseProduct );
         const noticeMock = jest.spyOn( utils, 'onSuccess' ).mockImplementation( () => {} );
-        const getMessageMock = jest.spyOn( message, 'getMessageByProductType' ).mockImplementation(() => {});
+		const getMessageMock = jest.spyOn( message, 'getMessageByProductType' ).mockImplementation((foo, bar) => {console.log(foo);console.log(bar);});
```

Comment:
We shouldn't be logging things to console. Did you leave these calls for some reason? You could just replace it with:
`const getMessageMock = jest.spyOn( message, 'getMessageByProductType' ).mockImplementation(() => {});`
Since we don't need any specific implementation for the test case.